flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcExecutionOptions.java

```diff
@@ -35,12 +35,18 @@
 	private final long batchIntervalMs;
 	private final int batchSize;
 	private final int maxRetries;
+	private final Integer parallelism;
 
-	private JdbcExecutionOptions(long batchIntervalMs, int batchSize, int maxRetries) {
+	private JdbcExecutionOptions(
+			long batchIntervalMs,
+			int batchSize,
+			int maxRetries,
+			Integer parallelism) {
```

Comment:
I'm not sure about adding this to `JdbcExecutionOptions`, because `JdbcExecutionOptions` is an public API for `JdbcSink`. For `JdbcSink#sink`, this may looks weird because the configured parallelism doesn't work as the returned object is `SinkFunction`.

Another option is adding the parallelism to `JdbcDmlOption` or `JdbcOption`, they are both internal class.